qelem/lib/data/remote/models/user_model.dart

```diff
@@ -5,33 +5,34 @@ UserModel userModelJson(String str) => UserModel.fromJson(json.decode(str));
 String userModelToJson(UserModel userModel) => json.encode(userModel.toJson());
 
 class UserModel {
-  String userName;
+  String username;
   String password;
   String firstName;
   String lastName;
   String role;
-  String profilePicture;
+  String? profilePhoto;
 
   UserModel(
-      {required this.userName,
+      {required this.username,
       required this.firstName,
       required this.lastName,
       required this.password,
       this.role = "MEMBER",
```

Comment:
Why does role has a default parameter? Like, we don't construct it.

Reply:
I didn't do this. Ask @Biruk-Tassew

Reply:
I believe it was @Biruk-Tassew  who did the models. Perhaps, we can ask him.